pkg/cli/build/build.go

```diff
@@ -199,6 +204,36 @@ func isImageDefinitionValid(ctx *image.Context) bool {
 	return true
 }
 
+func downloadKubernetesSELinuxPolicies(ctx *image.Context) error {
+	if ctx.ImageDefinition.Kubernetes.Version == "" {
+		return nil
+	}
+
+	configPath := filepath.Join(ctx.ImageConfigDir, "kubernetes", "config", "server.yaml")
+	config, err := kubernetes.ParseKubernetesConfig(configPath)
+	if err != nil {
+		return fmt.Errorf("parsing kubernetes server config: %w", err)
+	}
+
+	selinuxEnabled, _ := config["selinux"].(bool)
+	if !selinuxEnabled {
+		return nil
+	}
+
+	audit.Audit("SELinux is enabled in the Kubernetes configuration. " +
+		"The necessary RPM packages will be downloaded.")
+	zap.S().Info("SELinux mode requested for Kubernetes")
```

Comment:
If you use `AuditInfo`, you can skip the explicit zap log, it'll do it for you.

Reply:
I know but at the same time I didn't want the verbose UX logs in the log file. This is why I've went for "duplication" instead of using `AuditInfo`.

It's not a hard opinion though and I can change if you feel it's better.